cypress/e2e/Manage-candidate/Manage-candidate.ts

```diff
@@ -0,0 +1,128 @@
+import { Given, When, Then } from "@badeball/cypress-cucumber-preprocessor";
+import Admin from "../e2ePageObjects/Admin-page";
+import PIM from "../e2ePageObjects/PIM-page";
+import Recruitment from "../e2ePageObjects/Recruitment-page";
+import Logger from "../../support/Helpers/login-logout-helper";
+
+const AdminPage = new Admin();
+const PIMpage = new PIM();
+const RecruitmentPage = new Recruitment();
+const logger = new Logger();
+
+const HRMOrangeURLs = {
+  LOGIN_PAGE: "",
+};
+
+const ADMIN_INFO = {
+  USERNAME: "admin",
+  PASSWORD: "admin123",
+};
+
+const CANDIDATE_STATUS = {
+  SCHEDULED: "Interview Scheduled",
+  PASSED: "Interview Passed",
+  FAILED: "Interview Failed",
+};
+
+let employeeId: number;
+let jobTitleId: number;
+let addedCandidatePageUrl: string;
+
+// part 1 + 2: test case 1
+Given("The system have employee And job title And vacancy", () => {
+  cy.fixture("employee").as("employee");
+  cy.fixture("jobTitle").as("jobTitle");
+  cy.fixture("vacancy").as("vacancy");
+  cy.fixture("candidate").as("candidate");
+
+  cy.visit(HRMOrangeURLs.LOGIN_PAGE);
+  logger.passedLogin(ADMIN_INFO.USERNAME, ADMIN_INFO.PASSWORD);
+  cy.get("@employee").then((employee: any) => {
+    PIMpage.createEmployeeViaAPI(employee).then((response) => {
+      employeeId = response.data.empNumber;
+      PIMpage.createEmployeeWithLoginInfoViaAPI(employee, employeeId);
+    });
+  });
+
+  cy.get("@jobTitle").then((jobTitle) => {
+    AdminPage.createJobTitleViaAPI(jobTitle).then((response) => {
+      jobTitleId = response.data.id;
+    });
+  });
+
+  cy.get("@vacancy").then((vacancy) => {
+    RecruitmentPage.addVacancy(vacancy, jobTitleId, employeeId);
+  });
+
+  cy.get("@candidate").then((candidate) => {
+    RecruitmentPage.openAddCandidatePage();
+    RecruitmentPage.addCandidateViaUI(candidate).then((url) => {
+      addedCandidatePageUrl = url;
+    });
+    RecruitmentPage.shortListInitiatedCandidate();
+    RecruitmentPage.scheduleInterviewForCandidate();
+  });
+});
+
+Given('The system have candidate with status "Interview Scheduled"', () => {
+  RecruitmentPage.assertTheCurrentStatus(CANDIDATE_STATUS.SCHEDULED);
```

Comment:
in this step you put it in Given Why you assert inside it from the status?!
The assertion for the test case should be in Then step and should declared for the assert the test case 
here the system should candidate with Interview Scheduled as prepared data

Reply:
Okay , did you move the steps in the right place?

Reply:
Yes, of caurse